java/src/shared/model/game/Game.java

```diff
@@ -813,6 +819,15 @@ public void discardCards(int playerIndex, List<ResourceType> cards) throws Playe
         if (canDiscardCards(playerIndex) && this.turnTracker.canDiscard()) {
             playerManager.discardResourceType(playerIndex, cards);
         }
+        boolean change = true;
```

Comment:
I did, so I made this pull request am expression pod cdebug, where those
changes were after Danny pointed it out. So that once first, sorry I
extended it
On Mar 21, 2016 9:37 PM, "Derek Argueta" <notifications@github.com> wrote:

> In java/src/shared/model/game/Game.java
> <https://github.com/derekargueta/Settlers-of-Catan/pull/138#discussion_r56933463>
> :
>
> > @@ -813,6 +819,15 @@ public void discardCards(int playerIndex, List<ResourceType> cards) throws Playe
> >          if (canDiscardCards(playerIndex) && this.turnTracker.canDiscard()) {
> >              playerManager.discardResourceType(playerIndex, cards);
> >          }
> > +        boolean change = true;
>
> Please use a more descriptive variable name than change. It is difficult
> to tell exactly what the purpose of this code is.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/derekargueta/Settlers-of-Catan/pull/138/files/df673b6ee34e3d75d9101151f57bd3ddb2884e5c#r56933463>
>